subsys/bluetooth/host/scan.c

```diff
@@ -1246,6 +1246,7 @@ static void bt_hci_le_past_received_common(struct net_buf *buf)
 	sync_info.addr = &per_adv_sync->addr;
 	sync_info.sid = per_adv_sync->sid;
 	sync_info.service_data = sys_le16_to_cpu(evt->service_data);
+	sync_info.recv_enabled = false;
```

Comment:
This isn't the right solution

Whether this value should be `true` or `false`, depends on whether the `BT_LE_PER_ADV_SYNC_TRANSFER_OPT_REPORTING_INITIALLY_DISABLED` bit was set in the options when `bt_le_per_adv_sync_transfer_subscribe` was called. 

We don't have any way of storing that information currently, so this may be a bit tricky to implement. 

Effectively we should do the same as 
```c
	sync_info.recv_enabled =
		!atomic_test_bit(pending_per_adv_sync->flags,
				 BT_PER_ADV_SYNC_RECV_DISABLED);
```

but where the issue here is that we don't have `pending_per_adv_sync->flags`. 

To fix this issue, we need to store the `mode` from `bt_le_per_adv_sync_transfer_subscribe` in a way that we can retrieve it again here. 

This is a larger fix, and if you are not up for that, I understand (in which case we should create a GH issue for it). If you are not up for implementing a proper fix, then I suggest to default to `true` here instead of `false.

Reply:
Created the following GH issue:
https://github.com/zephyrproject-rtos/zephyr/issues/69927